urllib3/connection.py

```diff
@@ -57,9 +57,9 @@ class ConnectionError(Exception):
 }
 
 # When updating RECENT_DATE, move it to
-# within two years of the current date, and no
-# earlier than 6 months ago.
-RECENT_DATE = datetime.date(2016, 1, 1)
+# within two years of the current date, and not
+# less than 6 months ago.
```

Comment:
Should we add an example just for ultimate clarity, or a formula for generating this? For example

```
If today is 2018-01-01 then the RECENT_DATE shouldn't be any date after 2017-07-01. Roughly `datetime.date.today() - datetime.timedelta(days=6 * 31)`
```

Reply:
> If today is 2018-01-01 then the RECENT_DATE shouldn't be any date after 2017-07-01. Roughly `datetime.date.today() - datetime.timedelta(days=6 * 31)`

That's not my understanding, but rather: ```if Today is 2018-01-01, then RECENT_DATE SHOULD be any date on or after 2016-01-01 (today - 2 years) AND before 2017-07-01 (today - 6 months)```

Reply:
You're correct. I was more hoping to clarify the latter half because I still don't think it's entirely clear :) and what you have there looks actually perfect.

Reply:
Ok, I'll update the comment in the file with this example - then I think we should be clear for next time around we have to change it in about 18 months.